solution/ui/regulations/eregs-vite/src/components/subjects/SubjectChip.vue

```diff
@@ -38,7 +38,7 @@ defineProps({
         :to="{
             name: 'subjects',
             query: {
-                subjects: [subjectId.toString()],
+                subjects: subjectId.toString(),
```

Comment:
I had forgotten that we were explicitly setting the subjects query param as an array.  This is left over from when we were going to allow multiple subjects to be selected at the same time.